packages/common/src/allo/abis/allo-v2/Strategy.ts

```diff
@@ -0,0 +1,356 @@
+export default [
```

Comment:
Is this needed ? Since we are relying on the sdk -> can this be removed ?

Reply:
yes!! good catch!

Reply:
Just noticed that we use the abi in the test. We could just hardcode that event signature.